Libraries/LibIPC/Message.h

```diff
@@ -70,9 +96,16 @@ class Message {
     virtual int message_id() const = 0;
     virtual char const* message_name() const = 0;
     virtual ErrorOr<MessageBuffer> encode() const = 0;
+    u64 ipc_request_id() const { return m_request_id; }
+    constexpr bool is_response() const { return (message_id() % 2) == 0; }
 
 protected:
-    Message() = default;
-};
+    explicit Message(u64 request_id)
+        : m_request_id(request_id)
+    {
+    }
 
+private:
+    u64 m_request_id;
```

Comment:
Initialize plain types
(Yes, we do this even if there's a constructor to initialize it).